static/js/utils.js

```diff
@@ -131,6 +129,19 @@ var getLineNodeFromDOMInnerNode = function(originalNode, offsetOnBody) {
   return node;
 }
 
+// when the selection returns the body as the start node, the offset returns
+// which line the selection begins. The only exception of this rule is
+// when user deletes a selection that goes until the last line of the pad.
+// In this case, the line selected is 'offset - 1'
+var getLineSelectedUsingOffset = function (node, offset) {
+  var $lines = exports.getPadInner().find('div');
+  if (offset > $lines.length - 1) {
+    offset = offset - 1;
```

Comment:
se `offset` for `$lines.length + 10` esse teu código quebra. Melhor adotar uma solução mais geral e defensiva: se for maior que o no. de linhas, usar a última linha: `offset = $lines.length -1;`.